### DISCUSSION TITLE
Welcome to TallyArbiter Discussions!

### DISCUSSION THREAD

**Discussion by josephdadams** [josephdadams/TallyArbiter#237] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by LCalrissian** [comment]
Hello Mr. Adams and contributors, 

I'm testing Tally Arbiter with a Carbonite router (a fairly old legacy model, which only does TSL 3.1) and a newer Sony CCU - HDCU3100, which does TSL 5.0.  I'm using version 2.0.16 due to an open bug for the Carbonite in the version 3 releases, but the issue I'm encountering with the CCU happens with both versions.

The Sony CCU is receiving the Tally Arbiter messages and changing its tally lights, but then turns the lights off after three seconds.  My guess here is that the CCU expects the message to be repeated at least that often in order to keep the lights on.  There is no setting to manage this behavior on the CCU.    Is there any way to accomplish this in the software?  The Carbonite seems to send messages like PGM:True and PGM:False any time the tally is changed, so it would be a matter of repeating the last received True message for each address in use, until False is received.   

Thank you for your help.

**Reply by josephdadams** [reply]
Yes, the best fix for this would be to add TSL 5.0 output support to the "TSL Clients" Listener section in TA. It currently only sends TSL 3.1.

https://github.com/josephdadams/TallyArbiter/issues/211

Then you could enable the "1-sec updates" and it would be fine.

**Comment by LCalrissian** [comment]
Thank you for the response.  I will keep an eye on that feature request,
and hope that it is finished before I get a chance to learn javascript
programming :)  Thanks again for the project.

**Reply by josephdadams** [reply]
It’s been 3 years but hey you never know.
